packages/shopping/src/services/email.service.ts

```diff
@@ -0,0 +1,41 @@
+import {bind, BindingScope} from '@loopback/core';
+const nodeMailer = require('nodemailer');
```

Comment:
Types are fine. The error message could be more helpful.  Please install the types and try following changes, should work.  `npm i -D @types/nodemailer`

```suggestion
import { createTransport } from 'nodemailer';
```